src/libraries/System.Net.Sockets/ref/System.Net.Sockets.cs

```diff
@@ -443,6 +446,7 @@ public enum SocketAsyncOperation
         Send = 7,
         SendPackets = 8,
         SendTo = 9,
+        SendFile = 10,
```

Comment:
Is this needed or shall `Send` be used for _SendFileAsync_?

Reply:
Since we are not exposing the `SocketAsyncEventArgs`-based overload publicly there is no point extending the public API with this option, so let's remove it I guess. (Adding a comment about the workaround when `SocketAsyncOperation.Send` is being assigned.) @geoffkizer agreed?

Reply:
SocketAsyncEventArgs already has SendPackets, which is effectively a more general version of SendFile. So we shouldn't be adding SendFile to this enum.

I suppose it does raise the question of whether we should have a Task-based SendPacketsAsync method, though...